internal/k8s/controllers/config_controller_test.go

```diff
@@ -0,0 +1,335 @@
+/*
```

Comment:
Just so you know, no need to copy this manually. On new files, you can run inv bumplicense